WORKSPACE

```diff
@@ -11,6 +11,16 @@ git_repository(
     remote = "https://github.com/bazelbuild/bazel-skylib.git",
     tag = "0.2.0",
 )
+git_repository(
+    name = "io_bazel",
+    remote = "https://github.com/bazelbuild/bazel.git",
+    commit = "b749ab1f8549cdad0a574c356a57fe7181b07851",
```

Comment:
This is a dev version.

I agree with you in principle, I'd like to use a release. However, there isn't a release version of io_bazel that will work with skydoc yet.

So, can I leave a TODO here to adopt to a release tag once one is available? I'd like for users to be able to try this ASAP.

Reply:
I agree that aligning with a release is nice, but release tags aren't actually guaranteed to be reproducible so I'm not sure if the recommended practice is to use release tags (but I'm not sure)

Reply:
Hopefully release tags are reproducible and the upstream isn't reusing tags. That sounds as bad as force pushing commits that've already made it into circulation.

But even if it's a hash, it could still be a release hash annotated with the release name:

    commit = "..."   # release 0.1.2

Anyway, for this PR it's up to you whether to make a new branch or publish these changes to all users immediately.

Reply:
I would prefer to do what I've done here, marking this against a dev commit and starting some testing :) I've left a TODO to fix this later...